vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/exception/AdbCommandRejectedExceptionExt.kt

```diff
@@ -0,0 +1,10 @@
+package com.malinskiy.marathon.android.exception
+
+import com.android.ddmlib.AdbCommandRejectedException
+
+val deviceLostRegex = Regex("device \\'.+\\' not found")
```

Comment:
Wasn't `String.toRegex()` used throughout the project in similar definitions?

Reply:
True. I haven't look at the other parts of code, so I forgot :)